pkg/benchmark/image.go

```diff
@@ -37,16 +37,16 @@ const (
 )
 
 var defaultImageListingBenchmarkImagesAmd64 = []string{
-	"busybox:1.26.2-glibc",
-	"busybox:1-uclibc",
-	"busybox:1",
-	"busybox:1-glibc",
-	"busybox:1-musl",
+	"public.ecr.aws/docker/library/busybox:1.35.0-glibc",
+	"public.ecr.aws/docker/library/busybox:1-uclibc",
+	"public.ecr.aws/docker/library/busybox:1",
+	"public.ecr.aws/docker/library/busybox:1-glibc",
+	"public.ecr.aws/docker/library/busybox:1-musl",
 }
 var defaultImageListingBenchmarkImages = []string{
-	"busybox:1",
-	"busybox:1-glibc",
-	"busybox:1-musl",
+	"public.ecr.aws/docker/library/busybox:1",
+	"public.ecr.aws/docker/library/busybox:1-glibc",
+	"public.ecr.aws/docker/library/busybox:1-musl",
 }
```

Comment:
( I chose the AWS mirror because these tags are unavailable at `k8s.gcr.io/e2e-test-images` https://github.com/kubernetes/k8s.io/blob/19e888cf04b8da6ab1dd2d3999deb03e917f0dad/k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml#L29-L33 )

Reply:
Extract out the image prefix (`public.ecr.aws/docker/library/`) to a variable will make it easier to change in the future.

Reply:
done